packages/core/src/forms/control-group/control-group.element.ts

```diff
@@ -169,6 +172,11 @@ export class CdsInternalControlGroup extends LitElement {
     `;
   }
 
+  constructor() {
+    super();
+    ClarityIcons.addIcons(exclamationCircleIcon, checkCircleIcon);
+  }
```

Comment:
Missed one of the side effect issues with the angular cli where it drops the icon register if not in the component.